src/main/java/com/gmail/nossr50/events/skills/rupture/McMMOEntityDamageByRuptureEvent.java

```diff
@@ -10,12 +10,12 @@
 import java.util.HashMap;
 import java.util.Map;
 
-public class McMMOEntityDamageByRupture extends FakeEntityDamageByEntityEvent {
-	public McMMOEntityDamageByRupture(@NotNull Player damager, @NotNull Entity damagee, @NotNull DamageCause cause, double damage) {
+public class McMMOEntityDamageByRuptureEvent extends FakeEntityDamageByEntityEvent {
+	public McMMOEntityDamageByRuptureEvent(@NotNull Player damager, @NotNull Entity damagee, @NotNull DamageCause cause, double damage) {
```

Comment:
Should probably never construct it with Player over McMMOPlayer since McMMOPlayer should **always** be available